requirements/base.txt

```diff
@@ -6,7 +6,7 @@
 git+https://github.com/edx/XBlock.git@xblock-0.4.1#egg=XBlock==0.4.1
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@0.1.3#egg=edx-submissions==0.1.3
+git+https://github.com/edx/edx-submissions.git@efischer/soft_delete_sub#egg=edx-submissions
```

Comment:
Also just for sandbox consistency; this will be a proper edx-submissions release version before merging.